### PR TITLE
Milestone: Finished Design Teardown

### PR DESCRIPTION
**Pull request by newhorizon-tech**
The project has 

- A grayscale heatmap 
- Specific shades of gray based on the visual hierarchy of the elements
- Navigation bar
- Elements with rounded borders
- Custom fonts from Google Web Fonts 
- Used grids, floats, and flexbox for positioning and layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+      <p class="main-nav-logo tier-3">Main Logo</p>
+      <p class="nav-element tier-4">1</p>
+      <p class="nav-element tier-4">2</p>
+      <p class="nav-element tier-4">3</p>
+      <p class="nav-element tier-4">4</p>
+      <p class="nav-element tier-4">5</p>
+      <p class="nav-element tier-4">6</p>
+      <p class="nav-element tier-4">7</p>
+      <p class="search-nav tier-3">Search</p>
+    </nav>
```
Comment (SafaErden) [inline index.html]: - [x] It uses nav element with ul/li inside, however, students are allowed to used div as the main focus is heat map. So please use `ul/li` for nav items.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    <!-- <div class="test grid">
+      <div class="tier-1">One</div>
+      <div class="tier-2">Two</div>
+      <div class="tier-3">Three</div>
+      <div class="tier-4">Four</div>
+      <div class="tier-5">Five</div>
+    </div> -->
```
Comment (SafaErden) [inline index.html]: - [x] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+        <p>#lorem #lorem #lorem</p>
+      </div>
+      <div>
+        <div class="fake-pic"></div>
+        <p>Lorem ipsum dolor.</p>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+        <p>#lorem #lorem #lorem</p>
+      </div>
+    </article>
```
Comment (SafaErden) [inline index.html]: - [OPTIONAL] You can use the different shades of gray for background of each item for better visualization.
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+      </ul>
+      <div class="footer-content">
+      </div>
+    </footer>
```
Comment (SafaErden) [inline index.html]: - [x] The footer contains centered links, so please use proper HTML tags to create footer links and apply proper CSS attributes to center them.

image

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️

**Hi TEAM, your project is awesome. 👐 However, we need a little fix before we can proceed. Kindly find comments below**
##

**REQUIRED CHANGES**

- [x] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

* no repeating of the same rule

So please apply DRY CSS rules for your style document.

- [x] The other required changes are listed below, please kindly fix them.

##

**OPTIONAL CHANGES**

- [x] Please consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

##

**SUMMARY**

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

##

*Kindly make the changes and submit another code review when you are ready*


Questions or comments? Reach me on [Twitter](https://twitter.com/safaerden) or [LinkedIn](https://www.linkedin.com/in/safaerden/) 

Happy coding_ 👨‍💻 

**Safa ERDEN**

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

### I really like your project! Good job making the required changes and congrats team, you guys did an excellent job, you have my approval to go ahead and merge this pull request!

_**Clean and modular code, following the best practices and the design looks great, keep it up!**_ 🥇




## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**